### PR TITLE
Add list of possible values in yamlfile_value template

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Add list of possible options for OVAL related `yamlfile_value` fields.

#### Rationale:

- Clarify a bit what are the possible values and where they come from.

### CODE REVIEW

File: docs/manual/developer_guide.adoc [hunk 1]
```diff
@@ -1768,12 +1768,18 @@ yamlfile_value::
...
+** *entity_check* (link:https://github.com/OVALProject/Language/blob/master/docs/oval-common-schema.md#CheckEnumeration[CheckEnumeration]) - entity_check value for state's value_of, optional. If omitted, entity_check attribute would not be set and will be treated by OVAL as 'all'. +
+    Possible options are `all`, `at least one`, `none satisfy` and `only one`.
+** *check_existence* (link:https://github.com/OVALProject/Language/blob/master/docs/oval-common-schema.md#ExistenceEnumeration[ExistenceEnumeration]) - `check_existence` value for the `yamlfilecontent_test`, optional. If omitted, check_existence attribute will defalut to 'only_one_exists'. +
+   Possible options are `all_exist`, `any_exist`, `at_least_one_exists`, `none_exist`, `only_one_exists`.
 ** *values* - a list of dictionaries with values to check, where:
+**   *key* - the yaml key to check, optional. Used when the yamlpath expression yields a map. +
+     Note: Due to limitations in OVAL, uppercase letters should be escaped using `^`. For example, a yaml key `myCamelCase^key`, shouldbe expresed as `my\^camel^case\^^^key`.
```
Comment (yuumasato) [inline docs/manual/developer_guide.adoc:1777]: @evgenyz Feel free to improve this as you see fit.

**Comment by evgenyz** [comment]
Great, thanks! I'll merge and improve it in my PR

**Comment by evgenyz** [comment]
Actually, you know what, just remove the part about escaping. I'll add a separate PR with automatic escaping.

**Comment by yuumasato** [comment]
Cool, the failure was because of https://github.com/ComplianceAsCode/content/pull/6224

**Comment by evgenyz** [comment]
Nice, thanks!

**Pull request merged**